lib/twilio-ruby/version.rb

```diff
@@ -1,3 +1,3 @@
 module Twilio
-    VERSION = '6.9.1'
+    VERSION = '7.0.0'
```

Comment:
This should also be 7.0.0-rc.0. See [here](https://github.com/twilio/twilio-ruby/commit/a433a0239d87b2c5b0949c77268b3900c60185f1#diff-81996286d8704f7a7bc7c5e546c57431b655dcd8f55e93ae9e9874c4e5f01263)